app/src/main/java/com/example/accompany/CheckoutActivity.java

```diff
@@ -21,6 +22,11 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        try
+        {
+            this.getSupportActionBar().hide();
+        }
+        catch (NullPointerException e){}
```

Comment:
Lets log the exception to the console for now. I'm working on creating some logging so we can debug user exceptions